2001-3000/LOC2594Dayral.xml

```diff
@@ -47,7 +47,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="monastery">
+            <place type="monastery" sameAs="paths.places.112">
```

Comment:
I think we do the alignment to paths/pleiades/geonames etc in relations
`<relation name="skos:exactMatch" active="****" passive="http://****"/>`

Reply:
sameAs="wd:Q202661"